cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -413,9 +413,7 @@ def post_process_flow_cell(self, demux_results: DemuxResults) -> None:
             flow_cell_dir=demux_results.flow_cell.path, flow_cell_id=demux_results.flow_cell.id
         )
 
-    def finish_flow_cell(
-        self, flow_cell_name: str, bcl_converter: str, force: bool = False
-    ) -> None:
+    def finish_flow_cell(self, flow_cell_name: str, force: bool = False) -> None:
```

Comment:
I think there might be a need to manually override the conversion software, I think @ChrOertlin might have some idea about this

Reply:
Im going to revert, nice catch 👍

Reply:
reverted